pages/docs/manual/latest/import-from-export-to-js.mdx

```diff
@@ -57,30 +57,30 @@ Here's what the `external` does:
 
 By omitting the string argument to `module`, you bind to the whole JS module:
 
-<CodeTab labels={["ReScript", "JS Output (CommonJS)", "JS Output (ES6)"]}>
+<CodeTab labels={["ReScript", "JS Output (Module)", "JS Output (CommonJS)"]}>
 
 ```res example
 @module external leftPad: (string, int) => string = "./leftPad"
 let paddedResult = leftPad("hi", 5)
 ```
 ```js
-var LeftPad = require("./leftPad");
+import * as LeftPad from "./leftPad";
 var paddedResult = LeftPad("hi", 5);
```

Comment:
> ```res
> @module external leftPad: (string, int) => string = "./leftPad"
> let paddedResult = leftPad("hi", 5)
> ```

btw, this is a problematic example that only works with transpilers such as Babel or TypeScript (a.k.a synthetic default) and does not work with standard modules.